cli/src/main/java/com/liferay/blade/cli/command/WatchCommand.java

```diff
@@ -180,16 +408,6 @@ public void run() {
 
 		};
 
-		Runtime runtime = Runtime.getRuntime();
-
-		runtime.addShutdownHook(
-			new Thread(
-				() -> {
-					_uninstallBundles(projectOutputFiles);
-
-					_deleteWatchOutputFiles(projectOutputFiles);
-				}));
-
```

Comment:
Maybe this code should be added back and explicitly call `watchThread.stop()` or something along those lines, to ensure that thread actually stops cleanly?

Reply:
Nah, lets just bail! process is ending, it can clean up after itself! :)